specification/cognitiveservices/data-plane/TextAnalytics/preview/v3.0-preview.1/TextAnalytics.json

```diff
@@ -565,7 +565,7 @@
       "required": [
         "id",
         "sentiment",
-        "documentScores",
+        "confidenceScores",
```

Comment:
Shouldn't the folder name be `v3.1-preview.1` and not `v3.0-preview.1`?

Reply:
Yes, you are right, @laramume  also mentioned i should revert this. I reverted it.